src/foundry/client/data/documents/scene.d.ts

```diff
@@ -1,119 +1,88 @@
-     */
-    protected _view: this["data"]["active"];
-
+  class Scene extends ClientDocumentMixin(BaseScene) {
     /**
      * Track the viewed position of each scene (while in memory only, not persisted)
      * When switching back to a previously viewed scene, we can automatically pan to the previous position.
-     * @defaultValue `{}`
-     * @remarks This is intentionally public because it is used in Canvas._initializeCanvasPosition() and Canvas.pan()
+     * @internal
      */
-    _viewPosition: { x: number; y: number; scale: number } | {};
+    protected _viewPosition: CanvasViewPosition;
 
     /**
-     * A convenience accessor for whether the Scene is currently active
+     * Track whether the scene is the active view
+     * @internal
      */
-    get active(): this["data"]["active"];
+    protected _view: boolean;
 
     /**
-     * A convenience accessor for the background image of the Scene
+     * Determine the canvas dimensions this Scene would occupy, if rendered
      */
-    get img(): this["data"]["img"];
+    dimensions: SceneDimensions;
 
     /**
      * Provide a thumbnail image path used to represent this document.
      */
-    get thumbnail(): this["data"]["thumb"];
+    get thumbnail(): string;
+
+    /* -------------------------------------------- */
 
     /**
      * A convenience accessor for whether the Scene is currently viewed
      */
     get isView(): boolean;
 
-    /**
-     * A reference to the JournalEntry document associated with this Scene, or null
-     */
-    get journal(): InstanceType<ConfiguredDocumentClass<typeof JournalEntry>> | null;
-
-    /**
-     * A reference to the Playlist document for this Scene, or null
-     */
-    get playlist(): InstanceType<ConfiguredDocumentClass<typeof Playlist>> | null;
-
-    /**
-     * A reference to the PlaylistSound document which should automatically play for this Scene, if any
-     */
-    get playlistSound(): InstanceType<ConfiguredDocumentClass<typeof foundry.documents.BasePlaylistSound>> | null;
-
     /**
      * Set this scene as currently active
      * @returns A Promise which resolves to the current scene once it has been successfully activated
      */
-    activate(): Promise<this | undefined>;
+    activate(): Promise<Scene>;
+
+    /* -------------------------------------------- */
 
     /**
      * Set this scene as the current view
      */
-    view(): Promise<this | undefined>;
+    view(): Promise<Scene>;
 
-    /**
-     * @param createData - (default: `{}`)
-     * @param options    - (default: `{}`)
-     */
     override clone(
-      createData?: DeepPartial<SceneDataConstructorData | (SceneDataConstructorData & Record<string, unknown>)>,
-      options?: { save?: boolean; keepId?: boolean }
-    ): TemporaryDocument<this> | Promise<TemporaryDocument<this | undefined>>;
-
-    override prepareBaseData(): void;
+      data?: fields.SchemaField.AssignmentType<BaseScene.SchemaField["fields"], {}>,
```

Comment:
This can be shortened. `BaseScene.SchemaField["fields"]` is equivalent to `BaseScene.Schema`. And you can just simply use `BaseScene.UpdateData`.

Reply:
Updated

Reply:
It doesn't look like this was updated.